python.md

```diff
@@ -758,3 +758,55 @@ def test_some_longwinded_process(support_client, factory):
 ```
 
 You get the idea.
+
+
+## Documentation
+
+### READMEs
+
+Each directory within the domain layer should be modular and intuitive to use. A README can 
+optionally be placed in the root module directory and should detail specifics such as:
+
+- How is this module laid out?
+- How might someone use it?
+- What are the use cases?
+- How to extend behaviour (e.g. adding a new flow handler)
+- Who to contact if you run into errors
```

Comment:
I don't think we should be including names of particular developers in the code base. That kind of thing gets out of date. If we want to do something like that, it might make more sense to have it in a google doc.